### PR TITLE
Added Introduce.html and foods html file for clickable menu and remov…

### PR DESCRIPTION
**Pull request by SanazRabinia1**
…er bugs

### CODE REVIEW

File: WarriorDeliveryWebsite/DiningDelivery2/Breakfast.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
```
Comment (FrostedOrchard) [inline WarriorDeliveryWebsite/DiningDelivery2/Breakfast.html:1]: Delete file, will not be used

File: WarriorDeliveryWebsite/DiningDelivery2/Burger.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
```
Comment (FrostedOrchard) [inline WarriorDeliveryWebsite/DiningDelivery2/Burger.html:1]: delete file will not be used

File: WarriorDeliveryWebsite/DiningDelivery2/Breakfast.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+<html>
+    <head>
```
Comment (neelesh-wayne) [inline WarriorDeliveryWebsite/DiningDelivery2/Breakfast.html:3]: Remove unused files

**Review by FrostedOrchard — APPROVED** [review]
after pull, delete the 2 unnecessary files

**Pull request merged**